lib/Azure.ResourceManager.Authorization.deps.json

```diff
@@ -0,0 +1,347 @@
+{
```

Comment:
should this be a nuget dependency vs a local binary?  That is what we have done with the others

Reply:
This package isn't released, so it cannot be a nuget dependency.